apps/src/weblab2/helpers/aiTutorStructuredResponseHelper.ts

```diff
@@ -133,7 +133,16 @@ const getAnswerJsonSchemaAcceptReject = (): JsonObjectSchema => {
           additionalProperties: false,
         },
         description:
-          '`html`, `css`, or `js` fences. Limit to one language (html, css, or js) across the entire list. The list can be empty. Code should be formatted with appropriate newlines and indentation.  When providing modifications to student code, provide the entire contents of the file, the id and folderId of the file being updated. If the file is a new file and not currently in the the student code, then the id is "new", and the folderId is "0".The list can be empty. Code should be formatted with appropriate newlines and indentation. If the language is javascript or js, then the student will need to copy and paste this code into their project.',
+          '`html`, `css`, or `js` fences. Limit to one language (html, css, or js) across the entire list. ' +
+          'The list can be empty. Code should be formatted with appropriate newlines and indentation. ' +
+          'When providing modifications to a file in the student code, provide the entire contents of the file and include the file id and folderId of the file being updated. ' +
+          'For example, if a current student\'s file name is "demo.html", the file id is "2", and the folderId is "1", and the contents of the file has been updated, ' +
```

Comment:
I may end up just finding the file in current source manually in a follow-up because the model seems to still be sometimes inaccurate in returning the file id and folderId despite added context.